fastlane_core/lib/fastlane_core/project.rb

```diff
@@ -112,12 +112,21 @@ def project
       @project ||= Xcodeproj::Project.open(path)
     end
 
+    # xcodeproj doesn't include schemes that are under the .xcworkspace folder.
```

Comment:
Could you add a TODO with a link to https://github.com/CocoaPods/Xcodeproj/issues/557 if this is merged before we bump Xcodeproj please?

Reply:
It looks like it might take a day for my job to get me on the CLA, so I'll either put the TODO or just do the bump and call to load_schemes (depending on where xcodeproj is)

Reply:
@DanToml I added a "To Do" (this project's rubocop disallows TODO).  This is ready to merge -- I also added a PR to Xcodeproj -- if you would prefer to close this PR, and merge the xcode one -- I'd be happy to bump Xcodeproj here and add the load_schemes call.